tasks/s3roundtrip.py

```diff
@@ -28,13 +28,17 @@ def download(ctx, config):
     The context passed in should be identical to the context
     passed in to the main task.
     """
-    assert isinstance(config, list)
+    assert isinstance(config, dict)
```

Comment:
I looked a little further and I probably should make a change to the stuff in the finally: section of this routine.